### PR TITLE
Radius & track width config and setWheelSpeeds

### PR DESCRIPTION
**Pull request by calculusrobotics**
- Added radius and track width to config along with values for the robots we have
- implemented function that sets the wheel speeds of the drive base, used by the RAMSETE controller
- got rid of unused imports and methods
- also DriveConstants now needs to use the config to calculate constants based on config values

### CODE REVIEW

**Comment by cpostbitbuckets** [comment]
I"m not sure what happened, but the robot is REALLY slow now. It used to go 6k ticks per 100ms, now it goes 1k...

File: InfiniteRecharge/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -53,7 +52,7 @@
     @Override
     public void robotInit() {
         config = ConfigChooser.getConfig();
-        DriveConstants.setConfig(config);
+        DriveUtils.setConfig(config);
```
Comment (cpostbitbuckets) [inline InfiniteRecharge/src/main/java/frc/robot/Robot.java]: This is better, but does the DriveUtils class have to be static? If it requires an instance variable (config), why not just make it a member variable of the DriveSubsystem?

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveConstants.java [hunk 2]
```diff
@@ -97,23 +106,16 @@ public static double ticksP100ToIps(int ticksP100)
...
-    // S * 10 (100ms) / S
-    // rad * 1 rev/(2pi rad) * EPR ticks/rev
-    public static final double KV_100MS_PER_TICK /* WHY */ = KV_VOLT_S_PER_RAD * 10 / (2 * Math.PI) / DRIVE_MOTOR_NATIVE_TICKS_PER_REV / MAX_VOLTS;
 
-    public static final double KP_VOLT_S_PER_FT = 0.00332;
-    public static final double KP_VOLT_S_PER_RAD = KP_VOLT_S_PER_FT / 12 / (WHEEL_DIAMETER_INCHES / 2);
-    public static final double KP_100MS_PER_TICK_PERCENT /* WHY */ = KP_VOLT_S_PER_RAD * 10 / (2 * Math.PI) / DRIVE_MOTOR_NATIVE_TICKS_PER_REV / MAX_VOLTS;
+
+
+    public static DifferentialDriveKinematics KINEMATICS;
```
[the remark this answers is not in the document]
Comment (cpostbitbuckets) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveConstants.java]: One more thing on this specific variable, it's not constant so shouldn't be ALL caps or in DriveConstants, it's also never initialized. If you run the sim and click the Auto checkbox, you get an NPE (two values are actually null) in the call to the RamseteCommand super() method.

**Comment by cpostbitbuckets** [comment]
If we want to merge these changes without fixing auto, that's fine, but let's comment out this part of the DriveSubsystem that switches it to auto:

```java
        } else if (driverStation.isAutonomous()) {
            driveMethod = DriveMethod.AUTO; // please don't press any buttons during auto anyways :)))
        }
```

That way we can go into auto without it crashing the robot.

File: InfiniteRecharge/src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/Robot.java]: good point, I used static because DriveConstants was static but I guess it's not needed

**Comment by cpostbitbuckets** [comment]
This is much better, thanks.

**Pull request merged**